internal/action/handle.go

```diff
@@ -59,9 +59,7 @@ func SetCurrent(source *registry.RegistryDataSource, args []string) int {
 		logger.Error(internal.StringJoin("[Grm]: Can't found alias", name, "in your .nrmrc file. Please check it exist.", registry.Eol()))
 		return 1
 	}
-	flag, err := registry.WriteNpm(registry.RegsitryInfo{
-		Uri: uri,
-	})
+	flag, err := registry.WriteNpm(uri)
 	if flag {
 		logger.Success(internal.StringJoin("[Grm]: use", name, "success~", registry.Eol()))
 		return 0
```

Comment:
Why not use `if err != nil` ? Even though this style is silly, we should use a uniform style for our code. The use of `if flag` here also causes the call to writeNpm to `return 0` and `return 1` in a different style.

Reply:
You are right. May we can discuss how to unify the norms.Because in project. we have process state code. In my mind. Just like process state code we can provide a enum,
It look like 

eg:
```go

const (

NORMAL = 1 << iota
ABORT
)

```
And for other err. May `if err !=nil`. My mind don't have an answer for now :(

Reply:
Make sense to me.